cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,159 @@
+    cy.get("#firstName").type(user.firstName);
+    cy.get("#lastName").type(user.lastName);
+    cy.get("#userEmail").type(user.email);
+    cy.get("#age").type(user.age);
+    cy.get("#salary").type(user.salary);
+    cy.get("#department").type(user.department);
+    cy.get("#submit").click();
+
+    cy.get("#searchBox").type(user.firstName);
+    cy.get(".rt-td").should("contain", user.firstName);
+    cy.get('.rt-table').scrollIntoView().scrollTo('right');
+    cy.get("#edit-record-4 > svg").click();
+    cy.get("#firstName").clear().type(user.firstName + user.lastName);
+    cy.get("#lastName").clear().type(user.lastName);
+    cy.get("#userEmail").clear().type(user.email);
+    cy.get("#age").clear().type(user.age);
+    cy.get("#salary").clear().type(user.salary);
+    cy.get("#department").clear().type(user.department);
+    cy.get("#submit").click();
+
+    cy.get(".rt-td").should("contain", user.firstName + user.lastName);
+    cy.get(".rt-td").should("contain", user.lastName);
+    cy.get(".rt-td").should("contain", user.email);
+    cy.get(".rt-td").should("contain", user.age);
+    cy.get(".rt-td").should("contain", user.salary);
+    cy.get(".rt-td").should("contain", user.department);
+  });
+
+  it("should check search by all column values", () => {
+    cy.get("#addNewRecordButton").click();
+    cy.get("#firstName").type(user.firstName);
+    cy.get("#lastName").type(user.lastName);
+    cy.get("#userEmail").type(user.email);
+    cy.get("#age").type(user.age);
+    cy.get("#salary").type(user.salary);
+    cy.get("#department").type(user.department);
+    cy.get("#submit").click();
+
+    cy.get("#searchBox").type(user.firstName);
+    cy.get(".rt-td").should("contain", user.firstName);
+    cy.get("#searchBox").clear().type(user.lastName);
+    cy.get(".rt-td").should("contain", user.lastName);
+    cy.get("#searchBox").clear().type(user.age);
+    cy.get(".rt-td").should("contain", user.age);
+    cy.get("#searchBox").clear().type(user.email);
+    cy.get(".rt-td").should("contain", user.email);
+    cy.get("#searchBox").clear().type(user.salary);
+    cy.get(".rt-td").should("contain", user.salary);
+    cy.get("#searchBox").clear().type(user.department);
+    cy.get(".rt-td").should("contain", user.department);
+  });
+
+  it("should delete worker by Delete button", () => {
+    cy.get("#addNewRecordButton").click();
+    cy.get("#firstName").type(user.firstName);
+    cy.get("#lastName").type(user.lastName);
+    cy.get("#userEmail").type(user.email);
+    cy.get("#age").type(user.age);
+    cy.get("#salary").type(user.salary);
+    cy.get("#department").type(user.department);
+    cy.get("#submit").click();
+
+    cy.get("#delete-record-1").click();
+    cy.get(".rt-tbody").should("not.contain", user.firstName);
+    cy.get(".rt-tbody").should("not.contain", user.lastName);
+    cy.get(".rt-tbody").should("not.contain", user.email);
+  });
+
+  it("should delete workers by Delete buttons", () => {
```

Comment:
this test always fails and you need to delete all workers but not the 2

Reply:
done